samza-tools/src/main/resources/log4j.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
If we use the same packaging method as in samza-test:releaseTestJobs, we don't need to copy these resource files either.

Reply:
This resource file is not packaged. This file is  needed if you need logs when you run the tool using intellij. This is very similar to adding log4j.xml under test/resources e.g. samza/samza-log4j/src/test/resources/log4j.xml

Reply:
Got it.